book/history.md

```diff
@@ -243,24 +248,28 @@ which built [Netscape
 Navigator](https://en.wikipedia.org/wiki/Netscape_Navigator), the first
 _commercial browser_,[^commercial-browser] which launched in 1994.
 
-<div style="display: grid; grid-template-columns: repeat(2, 1fr);">
+<div class="grid" style="display: grid; grid-template-columns: repeat(2, 1fr);">
```

Comment:
What does this class name do?

Reply:
Reduce margins in the contained images. Probably better to have a different HTML structure...